content/en/docs/reference/glossary/endpoint.md

```diff
@@ -4,14 +4,19 @@ id: endpoints
 date: 2020-04-23
 full_link: 
 short_description: >
-  Endpoints track the IP addresses of Pods with matching Service selectors.
+  An endpoint of a Service is one of the Pods (or external servers) that implements the Service.
```

Comment:
despite is deprecated the API exist, so I think the old definition still applies

Reply:
I see, it was dicussed here https://github.com/kubernetes/website/pull/49831#discussion_r1965286613 , it seems is the preference to follow this path